app/notification.go

```diff
@@ -624,6 +624,11 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 func (a *App) userAllowsEmail(user *model.User, channelMemberNotificationProps model.StringMap, post *model.Post) bool {
 	userAllowsEmails := user.NotifyProps[model.EmailNotifyProp] != "false"
 
+	// if user is a bot account, then we do not send email
+	if user.IsBot {
+		return false
+	}
```

Comment:
Can the check even happen directly after the method starts?

Reply:
Hello, sorry I am not 100% sure on what you mean.

Are you referring to if the bot check can happen even after the method starts or if it even happens at all?

Reply:
I'm asking if the bot check can happen directly at the start of the method i.e. before `userAllowsEmails := user.NotifyProps[model.EmailNotifyProp] != "false"`

Reply:
Yes that can happen! Let me make that change on my end. Thank you for the clarification!